core/pom.xml

```diff
@@ -616,6 +621,50 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
SPARK-40593 and SPARK-41215 do some work to support Spark 3.4 to be compiled on CentOS6&7. I think the core module needs similar work, but may not be in this pr

Reply:
@WolverineJiang Are you interested in solving this problem after PR merge? Just like what you did in SPARK-41215

Reply:
> @WolverineJiang Are you interested in solving this problem after PR merge? Just like what you did in [SPARK-41215](https://issues.apache.org/jira/browse/SPARK-41215)

More than willing, I'll start in a few days.

Reply:
Thanks @WolverineJiang